### PR TITLE
Hh 2022 more updates

### PR DESCRIPTION
**Pull request by shubhsboson**

### CODE REVIEW

File: pages/faq.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ What about lodging and food?
 : See the [Venue page][venue] and the [Accommodation page][acco] for details.
 ^
 
+Can I work from hillhacks, any powercuts or internet issues ?
+: You sure can, generally there are no powecuts, expect for rains and storm situations.
```
Comment (ansuz) [inline pages/faq.md:77]: this should probably be 

> ..._except_ for rains...

Comment (shubhsboson) [reply pages/faq.md:77]: Yes , nice catch , Thanks

**Comment by shubhsboson** [comment]
Installed a spell checker extension fixing other typos as well

**Comment by shubhsboson** [comment]
Closing this as the changes have been merged.

**Pull request closed without merging**